include/libdnf/advisory/advisory_package.hpp

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright (C) 2021 Red Hat, Inc.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
```

Comment:
`version 2.1`

Reply:
I haven't updated this, it can be updated when its fixed everywhere as we discussed.